### PR TITLE
Updated version to 2.4

### PR DESCRIPTION
**Pull request by artemry-nv**

### CODE REVIEW

**Comment by artemry-nv** [comment]
@bureddy could you please review and merge?
Failed CI was disabled.

File: configure.ac [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 # See file LICENSE for terms.
 #
 AC_PREREQ([2.63])
 
 AC_COPYRIGHT([Copyright (c) 2019, NVIDIA CORPORATION & AFFILIATES. All rights reserved.])
 
-define([nccl_rdma_sharp_plugins_ver_major], 1)
-define([nccl_rdma_sharp_plugins_ver_minor], 0)
+define([nccl_rdma_sharp_plugins_ver_major], 2)
+define([nccl_rdma_sharp_plugins_ver_minor], 1)
```
Comment (bureddy) [inline configure.ac:11]: should we set minor number to 3,  as we are currently building 2.3 branch?

Comment (artemry-nv) [reply configure.ac:11]: Maybe change to `2.4` for `master` and `2.3` for `v2.3.x` branch (in separate PR)?

Comment (artemry-nv) [reply configure.ac]: Updated to 2.4

**Pull request merged**